shared/src/utils/stats/SurveyStatsCalculator.ts

```
@@ -0,0 +1,116 @@
+	): void {
+		const question = QUESTIONS_DICTIONARY.get(questionKey);
+		if (!question || !SUPPORTED_SURVEY_QUESTION_TYPES.includes(question.type)) return;
+
+		aggregatedData[questionnaire] = aggregatedData[questionnaire] || {};
+		const questionData =
+			aggregatedData[questionnaire][questionKey] ||
+			(aggregatedData[questionnaire][questionKey] = {
+				answers: {},
+				total: 0,
+				question,
+			});
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)